docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>I would suggest we add a comment in the next cell like:</p><p> </p><pre class="ql-syntax" spellcheck="false">The staging specification above only lists a single file name.  This will automatically be expanded to ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>added it and added examples for such directives</p>